qds_sdk/commands.py

```diff
@@ -87,10 +87,24 @@ def run(cls, **kwargs):
         Returns:
             Command object
         """
+        err_pointer, tmp_pointer, new_bytes = 0, 0, 0
         cmd = cls.create(**kwargs)
         while not Command.is_done(cmd.status):
             time.sleep(Qubole.poll_interval)
             cmd = cls.find(cmd.id)
+            if kwargs.get('print_logs_live', False):
+                log, err_length, tmp_length = cmd.get_log_partial(err_pointer, tmp_pointer)
```

Comment:
@msumit : Can you briefly describe your logic of updating the tmp and err pointers ? 
I do not have much idea about the how we fetch the logs from tmp and error files. 
It would help me in understanding your change better.